### PR TITLE
Make compatible with IE11

### PR DESCRIPTION
**Pull request by Artur-**
IE 11 does not support transform-style: preserve-3d so front and back
needs to be flipped individually.

### CODE REVIEW

**Comment by vpusher** [comment]
@Artur- IE11 does not support shadow dom v1, custom elements v1 and ES2015, so... 🙃

Did you transpiled it ?

**Comment by Artur-** [comment]
It needs to be transpiled for sure like all Polymer 2 elements and apps. Then the CSS and one or two JS methods needed some tweaking

**Comment by vpusher** [comment]
I know that but the project was originally created to provide some samples over the native APIs only. Nevertheless, I'm ok to extend the support in order to have a more 'ready to production' example. Consequently, I will have a look to start some automated build.

File: game-card.html [hunk 1]
```diff
@@ -361,7 +362,7 @@
             }
 
             _computeFigureImage(rank, symbol) {
-                var rankName = GameCard.FIGURES.includes(rank.toLowerCase()) && GameCard.RANKS[rank.toLowerCase()],
+                var rankName = GameCard.FIGURES.indexOf(rank.toLowerCase()) != -1 && GameCard.RANKS[rank.toLowerCase()],
```
Comment (vpusher) [inline game-card.html:365]: @Artur- why did you move from #includes to #indexOf ? is this because it is out of the ES2015 spec ?

**Review by vpusher — COMMENTED** [review]
Could you fix the **index.html** component page that is now broken and we are done here 🙂. It is possibly due to `<iron-component-page>` itself... dunno. If so, we would merge as is.

**Comment by Artur-** [comment]
Sorry, too much going on. IE11 does not support includes, that's why it was replaced. Was there still some index.html issue?

**Comment by vpusher** [comment]
@Artur- Ok. In more, `includes` comes from the ES2016/ES7 spec, removing it was the good thing to do. Concerning **index.html**, this si currently broken due to the `iron-component-page` which seems not yet ready for polymer 2. Thanks for your contribution
